examples/react/README.md

```diff
@@ -12,8 +12,9 @@ function App({ name }) {
 
 ```sh
 $ git clone https://github.com/eslint/eslint-plugin-markdown.git
-$ cd eslint-plugin-markdown/examples/react
+$ cd eslint-plugin-markdown
 $ npm install
+$ cd examples/react
 $ npm test
```

Comment:
It's necessary to install dependencies from the root because config uses the plugin module directly:

https://github.com/eslint/eslint-plugin-markdown/blob/e0da2214cc06fd441def1844135f3c9fef74e940/examples/react/eslint.config.mjs#L3